### PR TITLE
added rows directive to textarea

### PR DESCRIPTION
**Pull request by danieldeusing**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "12.5.2",
+  "version": "12.6.2",
```
Comment (Wallabeng) [inline package.json:3]: pls use 12.6.0

File: src/clr-addons/multilingual/abstract-multilingual.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ export abstract class ClrMultilingualAbstract extends ClrAbstractFormComponent {
   @Input('clrSelectedLang') selectedLang: string;
   @Input() readonly: string;
   @Input() maxlength: number;
+  @Input() rows: number;
```
Comment (Wallabeng) [inline src/clr-addons/multilingual/abstract-multilingual.ts:17]: pls move this to multilingual-textarea component, as it's not used in single line input

**Review by Wallabeng — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (danieldeusing) [reply package.json]: Bad mistake, done

File: src/clr-addons/multilingual/abstract-multilingual.ts (hunk 2 above)
Comment (danieldeusing) [reply src/clr-addons/multilingual/abstract-multilingual.ts]: done

**Review by Wallabeng — APPROVED** [review]

**Pull request merged**